addon/templates/components/polaris-card.hbs

```diff
@@ -1,4 +1,4 @@
-{{#if title}}
+{{#if (or title headerActions)}}
```

Comment:
Any reason this is wrapped in two conditions instead of doing

```handlebars
{{#if headerActions}}
  <!-- do the things in the first block -->
{{else if title}}
  <!-- do the things in the second block -->
{{/if}}
```

?

Reply:
Yes.

😝 

To expand, this came from mimicking the React implementation, which has a [`title || actions` condition](https://github.com/Shopify/polaris-react/blob/master/src/components/Card/Card.tsx#L53) in the main `Card` component which determines whether the `Header` subcomponent gets rendered 🤷‍♂️ I think this way is clearer when it comes to future maintenance but am open to changing things if you think there's value.